lib/AppPluginRF/index.js

```diff
@@ -26,10 +26,20 @@ const NpmCommands = require('../Modules/NpmCommands');
 class AppPluginRF extends AppPlugin {
 
 	async run() {
-    await NpmCommands.install({ save: true }, {
-      id: 'homey-rfdriver',
-      version: this._options.version,
```

Comment:
Note I have no idea what this `this.options.version` is, it is supposed to be set in `.homeyplugins.json` but it doesn't get set by the cli... Is it some user override so it doesn't try to install the latest version?

Reply:
I think it is an used property, but it would allow for installing a specific version of homey-rfdriver based on a `version` property in `.homeyplugins.json`.